tests/test_binaries.py

```diff
@@ -77,7 +96,7 @@ def _simple_cli_run(self, filename, contract=None, tx_limit=1):
         command.append(str(tx_limit))
         command.append(filename)
 
-        subprocess.check_call(command, stdout=subprocess.PIPE)
+        subprocess.check_call(command, stdout=subprocess.PIPE, cwd=working_dir)
```

Comment:
This is not really checking anything. Just that the process wont take > 10 mins I guess. We should try to add more checks here. Eventually

Reply:
It checks whether Manticore failed its execution or not. Initially I thought that the listed issues had to crash all Manticore and it returns exit code != 0.

But I guess that https://github.com/trailofbits/manticore/issues/676 did return 0 exit code, so I am confused a bit.